npm-shrinkwrap.json

```diff
@@ -8898,6 +8898,59 @@
         "strict-event-emitter-types": "^2.0.0",
         "ts-node": "^8.6.2",
         "typed-error": "^3.2.0"
+      },
+      "dependencies": {
```

Comment:
Interesting that this PR includes changes to `npm-shrinkwrap.json`, given that it does not make changes to dependencies in `package.json`. As if this PR was fixing the master branch's shrinkwrap, but running `npm i && npm dedupe && npm i` on the master branch does not produce any changes. 🤔

Reply:
Yeah this makes no sense, they don't get deduped and when I reset the changes and rerun those commands they don't come back, maybe I need to throw a `prune` in

Reply:
An npm prune doesn't solve it either :S